plugin/web/components/layout/WdLayout.tsx

```diff
@@ -0,0 +1,65 @@
+import { Layout, MenuProps } from "antd";
+import { MenuInfo } from "rc-menu/lib/interface";
+import React,{ memo, useMemo, useState } from "react";
+
+import "../../../node_modules/antd/dist/antd.less";
+import "../../styles/style.less";
+
+import { ReactComponent as DataBase } from "../UI/Icon/Icons/dataBase.svg";
+import { ReactComponent as Info } from "../UI/Icon/Icons/info.svg";
+import { ReactComponent as Share } from "../UI/Icon/Icons/share.svg";
+import { ReactComponent as Sliders } from "../UI/Icon/Icons/sliders.svg";
+import { ReactComponent as Template } from "../UI/Icon/Icons/template.svg";
+
+import WdContent from "./WdContent";
+import WdFooter from "./WdFooter";
+import WdHeader from "./WdHeader";
+
+const items: MenuProps["items"] = [
+  {
+    key: "mapData",
+    icon: <DataBase />,
+  },
+  {
+    key: "mapSetting",
+    icon: <Sliders />,
+  },
+  {
+    key: "shareNprint",
+    icon: <Share />,
+  },
+  {
+    key: "about",
+    icon: <Info />,
+  },
+  {
+    key: "template",
+    icon: <Template />,
+  },
+];
+export type Props = {
+  className?: string;
+  isInsideEditor: boolean;
```

Comment:
What is this isInsideEditor doing? Why does it slice up the items when in the editor and doesn't outside?

Reply:
I added this attribute so when we start the implementation we can pass it to define if we are inside reearth editor or out of it , so the tabs that appears in the side bar will be different